CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+## [5.0.1] - 2018-11-07
```

Comment:
`5.0.0`?

Reply:
I've already published a 5.0.0 so it needs to be bumped.